plugins/performance-lab/includes/admin/plugins.php

```diff
@@ -19,7 +19,7 @@
  * @return array{name: string, slug: string, short_description: string, requires: string|false, requires_php: string|false, requires_plugins: string[], download_link: string, version: string}|WP_Error Array of plugin data or WP_Error if failed.
  */
 function perflab_query_plugin_info( string $plugin_slug ) {
-	$transient_key = 'perflab_plugins_info';
+	$transient_key = 'perflab_plugins_info-v2';
```

Comment:
The transient key needs to be bumped since sites may already have downloaded bad plugin data with the old key.

Reply:
Seems like we should include a hash of the plugin version or something so that this type of bump would be unnecessary

Reply:
Oh, additionally, we should probably update the logic below so that if a slug is not found in the cache, it tries to request it from the API instead of returning a `WP_Error` right away.

Reply:
We could, but this is analogous to the `$wp_db_version` in core which doesn't necessarily need to change with each plugin release.

Reply:
> Oh, additionally, we should probably update the logic below so that if a slug is not found in the cache, it tries to request it from the API instead of returning a `WP_Error` right away.

Yeah, that's a good idea too.